lang/java/avro/src/test/java/org/apache/avro/reflect/TestReflect.java

```diff
@@ -1048,4 +1050,17 @@ public void testAvroDefault() {
           +"{\"name\":\"foo\",\"type\":\"int\",\"default\":1}]}");
   }
 
+  private static class NullableBytesTest {
+    @Nullable
+    byte[] bytes = "foo".getBytes();
+  }
+
+  @Test
+  public void testNullableByteArray() throws IOException {
```

Comment:
Since there are no asserts in this test, could you document in a comment how this test would fail without the fix? Thanks!